### ISSUE TITLE
Require at least one core content pack

### ISSUE THREAD

**Issue by Leland** [CleverRaven/Cataclysm-DDA#19413]
With #19260, we've moved the C-DDA content in a "core content pack," paving the way for future total conversion mods as envisioned by #17398.

At world creation, however, a user can remove the C-DDA content pack from the mod list, leaving the generated world content-less. Attempting to create a character within such a content-less world results in error after error notifying the player of missing IDs for terrain, furniture, etc.

I imagine we should probably require users select at least one core content pack at world generation.

Also, a thought: should it not only be "at least one core content pack," but also "at most one content pack"? Can/should users be able to load in multiple "games" into a single world?)

**Comment by Leland** [comment]
I finally got through all of the errors generated and was actually greeted with the character customization screen. Moving right to the "Scenario" pane resulted not in a debug error but a C-DDA-generated message saying "Nothing Found." This message cannot be closed. Pressing any button causes the message to pop up again. I had to quit the app to escape it.

It's a bit of an edge-case (a user starting in a world without scenarios) but surely we don't ever want a user to encounter a scenario where they have to force quit the game.
